core/app/models/spree/variant/pricer.rb

```diff
@@ -1,6 +1,15 @@
 module Spree
   class Variant
+    # This class is responsible for assigning a price to a variant.
+    # A variant can have multiple or even dynamic prices. The `price_for`
+    # method determines which price applies under the given circumstances.
+    #
     class Pricer
```

Comment:
Thomas says: Maybe another name that hints more at *choosing* or *retrieving* prices